code/game/objects/items/weapons/implants/implants/explosive.dm

```diff
@@ -103,8 +103,8 @@
 
 /obj/item/implant/explosive/exposed()
 	if(warning_message)
-		var/obj/item/radio/headset = get_global_headset()
-		headset.autosay(warning_message, "Anti Tampering System")
+		var/obj/item/radio/announcer = get_announcer(src)
```

Comment:
Should probably add a macro for get_announcer() check and autosay uses as per discord convo.